README.md

```diff
@@ -1,2 +1,145 @@
-# portfolio
-A portfolio webpage
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
```

Comment:
- [ ] Please add the `deployment` section to your `README` file because only except for the FAQ section or header, all others are required or mandatory to make your project looks more professional.